integration-testing.md

```diff
@@ -0,0 +1,606 @@
+    'admin-email' => \Magento\TestFramework\Bootstrap::ADMIN_EMAIL,
+    'admin-firstname' => \Magento\TestFramework\Bootstrap::ADMIN_FIRSTNAME,
+    'admin-lastname' => \Magento\TestFramework\Bootstrap::ADMIN_LASTNAME,
+    'amqp-host' => 'localhost',
+    'amqp-port' => '5672',
+    'amqp-user' => 'guest',
+    'amqp-password' => 'guest',
+];
+```
+
+Leave all the settings that do not start with `db-` and `amqp-` at their default values. You can include additional
+setup options available to the `setup:install` command in the test configuration file.
+
+<a href="#adjusting-the-phpunit-configuration-file"></a>
+### Adjusting the PHPUnit Configuration File
+
+The default integration test configuration is located in the `dev/tests/integration/phpunit.xml.dist` file. Without
+adjustments, this configuration runs all core integration tests, which is useful on a continuous integration server.
+
+When making adjustments to the configuration, copy the default file to `dev/tests/integration/phpunit.xml` (without
+the `.dist` suffix) and make your changes there. This ensures your changes won't be overwritten during Magento upgrades.
+
+There are many settings in the file. This guide will only describe three common adjustments. For more information about
+the available configuration settings, refer to
+the [PHPUnit documentation](https://phpunit.readthedocs.io/en/9.5/configuration.html) and comments in the default file.
+
+<a href="#tests-cleanup-constant"></a>
+#### TESTS_CLEANUP Constant
+
+The default value for the `TESTS_CLEANUP` constant is `enabled`. If set to `enabled`, the integration test framework
+cleans the test database and reinstalls Magento on every test run. This ensures any new modules are automatically picked
+up and any artifacts left behind by previous test runs are removed. It also causes the test framework to flush the test
+Magento configuration, the cache, and the code generation before executing any tests.
+
+The downside of setting `TEST_CLEANUP` to `enabled` is that the reinstallation of Magento takes time. The exact time
+depends on the host you are using to run the integration tests and the Magento version.
+
+During the development of new integration tests, where only a subset of the tests is executed repeatedly, the overhead
+of setting up a fresh execution environment for each run can quickly become a burden. In that case, the `TEST_CLEANUP`
+constant can be set to `disabled`. The test execution will start much quicker, but as a consequence, the developer must
+manually flush the cache and the database when needed.
+
+The integration test framework creates the temporary test files beneath the
+directory `dev/tests/integration/tmp/sandbox-*` (followed by a long hash ID). To force the test framework to regenerate
+the cache and the other files, remove the directory:
+
+```bash
+rm -r dev/tests/integration/tmp/sandbox-*
+```
+
+<a href="#rabbitmq-management-port-constant"></a>
```

Comment:
I don't think that mentioning the `RABBITMQ_MANAGEMENT_PORT` constant here adds value to the document.

Reply:
Kinda agree.